README.md

```diff
@@ -0,0 +1,55 @@
+![mailverse](logo.png)
+# Mailverse
+
+Mailverse is a new and modern email server written in NodeJS. It is designed to be fast, secure and easy to use.
+
+## Features
+
+- **SMTP** server with TLS support
+- **POP3** server with TLS support
+- 🔜 **IMAP** server with TLS support
+- 🆕 **Upstreams**: use an existing SMTP/POP server as relay and store your mail on own hardware. Unique to Mailverse
+- 🔜 **Web Interface** for managing your instance
+- 🔜 **Webmailer** with modern UI
+- **MariaDB / MySQL / File System** support for storing mail
+- 🔜 **Docker** support
```

Comment:
```suggestion
- **[SMTP]** **[POP3]** **[🔜IMAP]** server with TLS support
- 🆕 **Upstreams**: use an existing SMTP / POP server as relay and store your mail on own hardware.
Unique to Mailverse
- 🔜 **Web Interface** for managing your instance
- 🔜 **Webmailer** with modern UI
- **MariaDB / MySQL / File System** support for storing mail
- 🔜 **Docker** support
```
- **[SMTP]** **[POP3]** **[🔜IMAP]** server with TLS support
- 🆕 **Upstreams**: use an existing SMTP / POP server as relay and store your mail on own hardware.
Unique to Mailverse
- 🔜 **Web Interface** for managing your instance
- 🔜 **Webmailer** with modern UI
- **MariaDB / MySQL / File System** support for storing mail
- 🔜 **Docker** support